src/Dfc.CourseDirectory.Core/DataStore/CosmosDb/Models/Apprenticeship.cs

```diff
@@ -10,6 +10,12 @@ public class Apprenticeship
     {
         [JsonProperty("id")]
         public Guid Id { get; set; }
+        /// <summary>
```

Comment:
Given that this is a Cosmos specific model, perhaps this comment is unnecessary.

Reply:
I guess that depends who's reading it later. Personally I'd leave it as a breadcrumb for anyone who doesn't immediately make the jump.